README.md

```diff
@@ -192,21 +192,16 @@ Then run the following command from the root of the repository:
 
 #### Run the E2E tests on another network
 
-The E2E tests can be run on another network by implementing the `Network` interface in [`package network`](./tests/network/network.go). For example, the type `FujiNetwork` in [`example_fuji_network.go`](./tests/network/example_fuji_network.go) implements this interface, pointing to the [Amplify](https://subnets-test.avax.network/amplify), [Bulletin](https://subnets-test.avax.network/bulletin), and [Conduit](https://subnets-test.avax.network/conduit) Fuji subnets. After implementing this interface, you can run the E2E tests on this network by running a program such as:
-```go
-func main() {
-  // Register a failure handler that panics
-	gomega.RegisterFailHandler(func(message string, callerSkip ...int) {
-		panic(message)
-	})
-
-  // Run the test, composing it with the Network implementation
-  network := network.NewFujiNetwork()
-  defer network.CloseNetworkConnections()
-	tests.BasicOneWaySend(network)
-}
+The same E2E test flows can be executed against external network by setting the proper environment variables in `.env.testnet` and `.env`, and running the following commands:
```

Comment:
We check in `.env.example`, but `.env` is what's read by the tests. Can you add a step to copy `.env.example` into `.env`?